docs/_integration-tutorial-code/example-eslint-integration.js

```diff
@@ -0,0 +1,64 @@
+/** 
+ * @fileoverview An example of how to integrate ESLint into your own tool
+ * @author Ben Perlmutter
+ */
+
+const { ESLint } = require("eslint");
+
+function createESLintInstance(eslintConfig){
```

Comment:
I don't think this function is necessary because you're only ever using one branch of it.

Reply:
keeping the fxn b/c i think it's useful for the tutorial flow, but simplifying to not have the conditional, and just return the instance:

```js
// Create an instance of ESLint with the configuration passed to the function
function createESLintInstance(overrideConfig){
  return new ESLint({ useEslintrc: false, overrideConfig: overrideConfig });
}
```